### ISSUE TITLE
Terminal unresponsive while running ctop on remote server

### ISSUE THREAD

**Issue by phil-harmoniq** [microsoft/terminal#6792]
# Environment

```none
Windows build number: Microsoft Windows NT 10.0.18363.0
Windows Terminal version (if applicable): 1.0.1811.0

Any other software?
ctop and some running containers on a remote Ubuntu server
```

# Steps to reproduce
SSH into a linux server with containers running and ctop installed

# Expected behavior
Being able to scroll up/down and to the sides for more information

# Actual behavior
Terminal is unresponsive until ctop is closed with `q`

# Reference:
[ctop: Top-like interface for container metrics](https://github.com/bcicen/ctop)

**Comment by zadjii-msft** [comment]
Which ssh client are you using? `ssh.exe`, one that ships with WSL, some other one?

How are you trying to scroll in `ctop`? With the arrow keys? With the mouse wheel?

**Comment by phil-harmoniq** [comment]
I'm using the ssh client included with Git Bash. I'm trying to scroll with the arrow keys.

**Comment by zadjii-msft** [comment]
Hokay, so this is a long shot, but this might actually be #6737

**Comment by DHowett** [comment]
So, weird question ... if you use the SSH client included with Windows, does it suddenly work? You can run it directly from `C:\Windows\System32\OpenSSH\ssh.exe`.

**Comment by phil-harmoniq** [comment]
Interesting, it seems to work fine using Windows SSH client. I'd like to note that it used to work using the Git Bash SSH client.

**Comment by phil-harmoniq** [comment]
This issue seems to also happen in `htop`

**Comment by DHowett** [comment]
Thanks for the confirmation. That definitely makes it a /dup of #6737, which should be addressed by #6859
